### PR TITLE
Document test infrastructure

### PR DESCRIPTION
**Pull request by gjsjohnmurray**
This PR tweaks the Docker-based testing infrastructure and documents it in a new page in the Contributing section.

### CODE REVIEW

File: docs/src/pages/contributing/testing.mdx [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+yarn run v1.22.15
+$ bash ./docker/mysql/prepare.sh
+Done in 0.10s.
+Creating network "docker_default" with the default driver
+Pulling mssql (mcr.microsoft.com/mssql/server:latest)...
+latest: Pulling from mssql/server
+85b353ff0ac5: Pull complete
+a561de296022: Pull complete
+274645b68430: Pull complete
+Digest: sha256:ca2d56561e6cc50b175d97aeffc0efe33d661719ab94b826684b3ac5bff41d58
```
Comment (mtxr) [inline docs/src/pages/contributing/testing.mdx]: Not sure if worthy keeping the digest and some portion of those hashed stuff here. They might change quite often as we move forward and I'm afraid new comers may get confused expecting to see those. What do you think if we omit some lines replacing by ...?

Comment (gjsjohnmurray) [reply docs/src/pages/contributing/testing.mdx]: Good point. I have pushed a change.

**Review by mtxr — APPROVED** [review]

**Comment by gjsjohnmurray** [comment]
Thanks for approving. Please merge it. I'm not being offered the option to do than because not all required checks have passed.

**Comment by gjsjohnmurray** [comment]
@mtxr thanks for merging. I was working out how to use the `gh api` command to manually set the two [commit statuses](https://docs.github.com/en/rest/commits/statuses) that unlock the ability to merge. Managed to do it for #929. I'd like to understand the normal procedure for achieving this. Perhaps email me directly about it?

**Pull request merged**